index.js

```diff
@@ -1,18 +1,17 @@
 var quadraticElement = document.getElementById("quadratic");
 var solutionElement = document.getElementById("solution");
-// (ax + b)(cx + d)
```

Comment:
I personally think these comments are necessary

Reply:
i personally think u should pipe down